doc/articles/Uno-UI-Performance.md

```diff
@@ -1,6 +1,6 @@
 # Uno.UI - Performance
 
-This article lists a number of performance tips to optimize your Uno Platform application. 
```

Comment:
ℹ️ Apologies. Looks like my auto-formatting got this file. If you want me to revert the formatting changes throughout, just let me know.

Reply:
Resolving this; as it had to be resolved for merging.